docs/standards/smirnoff.md

```diff
@@ -364,13 +368,28 @@ Later revisions will also provide support for special interactions using the `<A
 
 Electrostatic interactions are specified via the `<Electrostatics>` tag.
 ```XML
-<Electrostatics version="0.3" method="PME" scale12="0.0" scale13="0.0" scale14="0.833333" scale15="1.0"/>
+<Electrostatics version="0.4" periodic_potential="Ewald3D-ConductingBoundary" nonperiodic_potential="Coulomb" exception_potential="Coulomb" scale12="0.0" scale13="0.0" scale14="0.833333" scale15="1.0"/>
 ```
-The `method` attribute specifies the manner in which electrostatic interactions are to be computed:
 
-* `PME` - [particle mesh Ewald](https://docs.openmm.org/latest/userguide/theory.html#coulomb-interaction-with-particle-mesh-ewald) should be used (DEFAULT); can only apply to periodic systems
-* `reaction-field` - [reaction-field electrostatics](https://docs.openmm.org/latest/userguide/theory.html#coulomb-interaction-with-cutoff) should be used; can only apply to periodic systems
-* `Coulomb` - direct Coulomb interactions (with no reaction-field attenuation) should be used
+Some methods for computing electrostatic interactions are not valid for periodic systems, so
+separate methods must be specified for periodic (`periodic_potential`) and non-periodic
+(`nonperiodic_potential`) systems.
+
+The `periodic_potential` attribute specifies the manner in which electrostatic interactions are to be computed in periodic systems. Allowed values are:
+
+* `Ewald3D-ConductingBoundary` (default) denotes that a method like [particle mesh Ewald](https://docs.openmm.org/latest/userguide/theory.html#coulomb-interaction-with-particle-mesh-ewald) should be used
+* `Coulomb` denotes that the standard Coulomb potential should be used with specified cutoff `cutoff` and optionally switch width `switch_width`
```

Comment:
```suggestion
* `Coulomb` denotes that the standard Coulomb potential should be used with no cutoff or reaction-field attenuation
```

Reply:
Is this right? I'm under the impression that "coulomb with no cutoff" is not possible for _periodic_ systems, since there are an infinite number of periodic particles to interact with. So I guess I'm assuming that "coulomb" here means "coulomb with a cutoff"

Reply:
I agree that Coulomb with no cutoff (in periodic systems) is not typically something someone would want, and if you WERE going to do as Simon suggests you would probably want to SAY that that's the convention being used.

I think I'd (somewhat) advocate for taking Coulomb out entirely for periodic systems, but I'm also OK with amending this by specifying Simon's resolution (calculate only the nearest interaction).

Reply:
(Once this issue is resolved I'm ready to see this PR merged.)

Reply:
+1 for taking the Coulomb option out entirely for periodic systems

Reply:
Ok, I've taken out the coulomb option out entirely for periodic systems